sale_product_pricelist_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Product Pricelist Operating Unit",
+    "version": "17.0.1.0.0",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
```

Comment:
```suggestion
    "license": "AGPL-3",
```

Reply:
It should be the same license as operating_unit